community/blog/2023-02 Rebranding Arcus for maximum adaptation/README.md

```diff
@@ -0,0 +1,36 @@
+# Rebranding Arcus for maximum adaptation
+Arcus has received many updates regarding the branding of each component's feature documentation. Just recently, we took the final step in the rebranding of Arcus: a whole new general Arcus landing zone.
+
+## Central place for everything Arcus
+We already had a general Arcus site, but the problem was that it did not contain enough information to have almost any reason to exist. The only reason we had it was to have a central location to direct people to and from where they could be redirected to specific Arcus components. With the rebranding of Arcus' style we also took it upon us to rethink our central place for everything Arcus. Instead of it being a meager 'forwarding' site, it is now the place to be to find any general information on Arcus. We have listened to the comments of the people using Arcus and the people hesitant to use Arcus. With this knowledge, the website has received a whole new makeover that will maximize adaptation.
+
+🎉 Navigate to [the new website](https://arcus-azure.net/)!
+
+## The 'What', 'Why', and 'How' questions
+It was the perfect time, during the creation of the new site with new contents, to reflect on what the general Arcus website should say and inform on. As Arcus is already available and used for several years, we know what the common issues are for not adapting Arcus. It was therefore our goal to make this new site a central place to answer all these common questions.
```

Comment:
```suggestion
It was the perfect time, during the creation of the new site with new contents, to reflect on what the general Arcus website should say and inform on. As Arcus is already available and used for several years, we know what the common issues are for not adopting Arcus. It was therefore our goal to make this new site a central place to answer all these common questions.
```
I think you mean 'adopting' (opnemen, gebruiken) instead of adapting (aanpassen) ?

Reply:
Aha, yes, 😉 , thx for this! 👍